web/client/components/data/identify/viewers/row/RowViewer.jsx

```diff
@@ -42,6 +42,7 @@ function RowViewer({
     const Row = layerRowViewer || component || PropertiesViewer;
     return (
         <Row
+            fields={layer?.fields}
             {...omit(feature.properties, ['ref'])}
```

Comment:
if properties contains a fields key the will override the prop.fields, is it expected?

Reply:
good point. Going to move it below. Anyway if properties are varying, maybe we should avoid spreading them in the future, to avoid overlapping in fields.